db/executer.go

```diff
@@ -22,23 +22,21 @@ import (
 
 	"database/sql"
 	"fmt"
+	"strings"
+
 	"github.com/jinzhu/gorm"
 	_ "github.com/jinzhu/gorm/dialects/postgres"
-	"strings"
 )
 
 type (
 	finder interface {
 		find(out interface{}, where ...interface{}) error
```

Comment:
should we make this consistent with the mulitinserter and change `interface{}` to `Record`?

Reply:
changed in e2da5d7, thanks!